### ISSUE TITLE
Rounded atelectasis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6710]
**Preferred term label:**
Rounded atelectasis

**Synonyms**
Folded lung syndrome
Helical atelectasis
Blesovsky syndrome
pleural pseudotumor
Pleuroma
Comet tail sign

**Definition (free text, please give PubMed ID)**
Rounded atelectasis is rounded collapsed lung associated with invaginated fibrotic pleura and thickened and fibrotic interlobular septa. Most frequently, it is the consequence of an asbestos-induced exudative pleu- ral effusion with resultant pleural scarring, but it may occur with any cause of pleural fibrosis. On chest radiographs, rounded atelectasis appears as a mass abutting a pleural surface, usually in the posterior part of a lower lobe. Distorted vessels have a curvilinear disposition as they converge
on the mass (the comet tail sign). The degree of lobar retraction depends on the volume of atelectatic lung. It is almost invariably associated with other signs of pleural fibrosis (eg, blunting of costophrenic angle). CT is more sensitive for the detection and display of the characteristic features of rounded atelectasis. An additional sign is homogeneous uptake of contrast medium in the atelectatic lung. 

PMID:18195376

Comment
See Figure 65 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
